crm_lead_to_event_registration/wizard/crm_lead_event_pick.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class WizardModel(models.TransientModel):
+    _name = "crm.lead.event.pick"
+
+    lead_id = fields.Many2one(
+        "crm.lead",
```

Comment:
This is a good practice BTW because parameters order could change in future (not in Odoo, of course). If you set by keywords, changing order will not affect to this call.